### DISCUSSION TITLE
FulfillmentInbound

### DISCUSSION THREAD

**Discussion by manibha-jain** [saleweaver/python-amazon-sp-api#234] [Q&A]
I am trying to use get_shipments in  FulfillmentInbound and it is giving error 
`Missing or invalid request parameters: [QueryType]`
What is the syntax to pass QueryType in get_shipments() ?

**Comment by maonogit** [comment]
![image](https://user-images.githubusercontent.com/100677439/178237232-04ad0356-c7c4-4a75-b3e8-64323f38566f.png)
